src/main/resources/database/changes/release-10.60.0/update_categories_to_use_both_b_and_bi_cases.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE casesvc.category
+SET oldcasesampleunittypes = 'B,BI'
+WHERE categorypk IN ('COLLECTION_INSTRUMENT_DOWNLOADED', 'COLLECTION_INSTRUMENT_ERROR',
+'UNSUCCESSFUL_RESPONSE_UPLOAD', 'SUCCESSFUL_RESPONSE_UPLOAD');
```

Comment:
`SUCCESSFUL_RESPONSE_UPLOAD` has `H` as an old type currently. This will remove that which we shouldn't do